pkgs/applications/misc/peaclock/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, cmake, libpthreadstubs, icu }:
+
+stdenv.mkDerivation rec {
+  pname = "peaclock";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "octobanana";
+    repo = pname;
+    rev = version;
+    sha256 = "1582vgslhpgbvcd7ipgf1d1razrvgpq1f93q069yr2bbk6xn8i16";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ libpthreadstubs icu ];
+
+  enableParallelBuilding = true;
```

Comment:
This is not required with CMake, as it is enabled by default. See [this section of the manual](https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-attributes) for more information.